Makefile

```diff
@@ -281,6 +293,19 @@ lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
 generate-mocks: mockery ## generate mock objects
 	PATH=$(LOCALBIN):$(PATH) go generate ./...
 
+
+.PHONY: generate-api-docs
+generate-api-docs: gen-crd-api-reference-docs ## generate api documentation
+	$(GEN_CRD_API_REFERENCE_DOCS) -api-dir=./api/v1alpha1 -config=${CURDIR}/hack/api-docs/config.json \
+	-template-dir=${CURDIR}/hack/api-docs/templates -out-file=$(BUILDDIR)/api-reference.html
+	$(CONTAINER_TOOL) run --rm --volume "`pwd`:/data:Z" pandoc/minimal -f html -t markdown_strict \
```

Comment:
if we dont want to rely on docker image of pandoc (hosted at dockerhub) 

an alternative is to:
1. see if we can download binary assuming no dependencies
2. use https://github.com/suntong/html2md

Reply:
I think docker image is ok